Services/UI/OJS.Services.Ui.Models/Contests/ContestProblemServiceModel.cs

```diff
@@ -83,13 +73,13 @@ public void RegisterMappings(IProfileExpression configuration) =>
                 .ForMember(d => d.UserHasAdminRights, opt => opt.Ignore())
                 .ForMember(
                     d => d.AllowedSubmissionTypes,
-                    opt => opt.MapFrom(s => s.SubmissionTypesInProblems.Select(st => st.SubmissionType)))
+                    opt => opt.MapFrom(s => s.SubmissionTypesInProblems))
                 .ForMember(
                     d => d.Points,
                     opt => opt.MapFrom(s => 0))
                 .ForMember(
                     d => d.MemoryLimit,
-                    opt => opt.MapFrom(s => (double)s.MemoryLimit))
+                    opt => opt.MapFrom(s => (double?)s.MemoryLimit / 1024 / 1024))
```

Comment:
Can we do the conversion on the front end?

Reply:
Fixed